example/src/App.svelte

```diff
@@ -20,18 +20,19 @@
   bind:value={name} />
 <br />
 
-<h1>{$_.title('greeting.message', { name })}</h1>
+<h1>{$_.title('greeting.message', { values: { name } })}</h1>
```

Comment:
Maybe change the signature to be the same as before, but in the implementation of title, add the wrapper with values:
`$_.title('greeting.message', { name } )`

Reply:
Having different signatures for the main format method and the utilities method is kind of counter intuitive 🤔 I have thought about keeping the current signature and do some destructuring magic to separate what's what, but I'm not sure about name collision in this case... Imagine having a message `current language is {locale}`, when I format it with `$_('message', { locale: 'pt-BR'})` it would try to force a locale with that value rather than inserting it in the message.